specifyweb/specify/views.py

```diff
@@ -768,3 +768,145 @@ def abort_merge_task(request, merge_id: int) -> http.HttpResponse:
 
     else:
         return http.HttpResponse(f'Task {merge.taskid} is not running and cannot be aborted.')
+
+
+@openapi(schema={
+    "post": {
+        "requestBody": {
+            "required": True,
+            "content": {
+                "application/json": {
+                    "schema": {
+                        "type": "object",
+                        "properties": {
+                            "columnHeaders": {
```

Comment:
(optional) add more documentation or example of what column headers are expected